src/containers/Contactus.jsx

```diff
@@ -0,0 +1,230 @@
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import { useRef } from 'react';
+import emailjs from 'emailjs-com';
+import contactus from '../assets/contactus.svg';
+
+function Contactus() {
+  const modelRef = useRef();
+
+  const sendEmail = (values) => {
+    console.log(modelRef);
+    emailjs
+      .send('service_va0wqwg', 'template_l4l91a5', values, 'JrpFs7BlCv1nsgafI')
+      .then(
+        () => {
+          modelRef.current.style.display = 'block';
+          modelRef.current.childNodes[0].style.display = 'block';
+          modelRef.current.childNodes[1].style.display = 'none';
+        },
+        () => {
+          modelRef.current.style.display = 'block';
+          modelRef.current.childNodes[1].style.display = 'block';
+          modelRef.current.childNodes[0].style.display = 'none';
+        },
+      );
+  };
+
+  return (
+    <section className="text-gray-600 body-font relative">
+      <h1 className="text-gray-400 text-5xl p-5 font-medium">CONTACT US.</h1>
+      <h2 className="text-3xl font-medium drop-shadow-md px-10">
+        Please get in touch and our expert team will answer all your questions.
+      </h2>
+      <div className="container px-5 py-2 mx-auto flex sm:flex-nowrap flex-wrap flex-col md:flex-row items-center">
+        <div className="lg:w-2/3 md:w-1/2 w-full">
+          <img src={contactus} alt="Contact Us" />
+        </div>
+        <div className="rounded lg:w-1/3 md:w-1/2 bg-white flex flex-col md:ml-auto w-full md:py-8 mt-8 md:mt-0 justify-center p-4">
+          <Formik
+            initialValues={{ name: '', email: '', message: '' }}
+            validate={(values) => {
+              const errors = {};
+              if (values.email === '') {
+                errors.email = 'Email is required';
+              } else if (!/^[^\s@]+@[^\s@]+\.[^\s@]{2,}$/i.test(values.email)) {
+                errors.email = 'Invalid email address format';
+              }
+              if (values.name === '') {
+                errors.name = 'Name is required';
+              }
+              if (values.message === '') {
+                errors.message = 'Bodys Message is required';
```

Comment:
Let's change the error message for the message input to something like "Please add a message at least one character long".